lib/entities/context/DutchQuoteContext.ts

```diff
@@ -185,13 +199,16 @@ export class DutchQuoteContext implements QuoteContext {
     }
 
     const syntheticStatus = await this.syntheticStatusProvider.getStatus(this.request.info);
-    // if the useSyntheticQuotes override is not set, and the request is not eligible for synthetic, return null
-    if (!this.request.config.useSyntheticQuotes && !syntheticStatus.syntheticEnabled) {
+    const chainId = classicQuote.request.info.tokenInChainId;
+    const quoteConfig = ChainConfigManager.getQuoteConfig(chainId, this.request.routingType);
+    // if the useSyntheticQuotes override is not set by client or server and we're not skipping RFQ, return null
+    // if we are skipping RFQ, we need a synthetic quote
+    if (!this.request.config.useSyntheticQuotes && !syntheticStatus.syntheticEnabled && !quoteConfig.skipRFQ) {
```

Comment:
So if the client sets `useSyntheticQuotes` to false but skips RFQ we will end up returning a synthetic quote? That seems a bit weird to me, maybe we should add joi validation against that

Reply:
Yeah I can see how it'd be weird. Should we return no quotes instead if client sets `useSyntheticQuotes` to false and we have `skipRFQ` set?

Reply:
We chatted more offline. Here's my latest thinking:
`useSyntheticQuotes` set to true means the client wants to force synth quotes. `useSyntheticQuotes` set to false doesn't mean that they don't want synth quotes - it implies that we should default to whatever we normally would have done. 

Similarly, if `skipRFQ` is set to false it doesn't mean anything - only setting it to true should force a synth quote. 

So we should only skip synth quotes if all of the following is true:
- We aren't skipping RFQ
- The user isn't asking for a synth quote
- The synth status endpoint isn't returning true